ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -292,6 +292,7 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
                 unit={unit}
                 grid={gridOverrides}
                 tooltipConfig={{ wrapLabels: true }}
+                syncGroup="default-panel-group"
```

Comment:
Eventually, this will be set only if the dashboard spec has `dashboard.spec.settings.graphtoolTip === 'shared'`.